Ricochet/Assets/_Scenes/TransitionScenes/MainMenu.unity

```diff
@@ -309,6 +309,8 @@ CanvasRenderer:
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
   m_GameObject: {fileID: 80268780}
+<<<<<<< HEAD
```

Comment:
Merge headers need to be removed. Main menu isn't working with them in the file.